docs/SOPs/Managed_access/Pre-submission_SOP.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Pre-submission SOP
+parent: Managed access
+grand_parent: SOPs
+nav_order: 3
+last_modified_date: 30/05/2024
+---
+
+<script src="https://kit.fontawesome.com/fc66878563.js" crossorigin="anonymous"></script>
+
+# Managed access dataset - Pre-submission SOP
+Refer to this guide to prepare for the ingestion of a managed access dataset once the DCA has been signed.
+
+## Register the project in DUOS system
+
+A copy of the managed access project will have to be created in [DUOS](https://duos.org/) to control access to the dataset once it’s available on the Data Portal.
+
+1. Email Jonathan at jlawson@broadinstitute.org with the [DUOS project information fields](https://docs.google.com/document/d/18pzeKafFQZ0rhqrb2DLOCciHUu4On4g2rtgnqygIfTE/edit#heading=h.674bm0dao5mr), using the information from the DCA agreement. This is mainly information about the dataset, the only personal data shared with the Broad is the PI’s name.
+2. Register in ingest the project’s DUOS id provided by Jonathan
```

Comment:
ingest --> HCA Data Repository Ingest Service 

Jonathan --> the DUOS system representative